packages/graphqlsp/src/diagnostics.ts

```diff
@@ -247,6 +250,26 @@ export function getGraphQLDiagnostics(
           };
         }
 
+        const hash = callExpression.arguments[0].getText().slice(1, -1);
+        if (hash.startsWith('sha256:')) {
+          const upToDateHash = `sha256:${generateHashForDocument(
```

Comment:
This may compose into `sha256:undefined` which then likely always issues the warning instead of bailing out

Reply:
we have basically evaluated all the pre-conditions for this not to happen though